test/remote-httputility.cpp

```diff
@@ -0,0 +1,77 @@
+/* Icinga 2 | (c) 2025 Icinga GmbH | GPLv2+ */
+
+#include <BoostTestTargetConfig.h>
+#include "remote/httputility.hpp"
+#include "test/icingaapplication-fixture.hpp"
+
+using namespace icinga;
+
+BOOST_AUTO_TEST_SUITE(remote_httputility)
+
+BOOST_FIXTURE_TEST_CASE(IsValidHeaderName, IcingaApplicationFixture)
```

Comment:
You don't need the `IcingaApplicationFixture` here. It's always globally registered:
https://github.com/Icinga/icinga2/blob/4e840cffff8e87bdc0e27c7b787f4876e1393961/test/icingaapplication-fixture.cpp#L32